product_pack/readme/USAGE.rst

```diff
@@ -1,10 +1,10 @@
 To use this module, you need to:
 
-#. Go to *Sales > Products > Products*, create a product and set "Is Pack?".
-#. Then set the products has to include in the product pack of the product.
+#. Go to *Sales > Products > Products*, create a product and set *Is Pack?*.
```

Comment:
Don't change italics here.

Reply:
Done, but IMHO italic is easier to read, because it is used previously (I come from typography :-)